cypress-tests/cypress/support/commands.js

```diff
@@ -393,16 +437,24 @@ Cypress.Commands.add("voidCallTest", (requestBody, globalState) => {
       "Content-Type": "application/json",
       "api-key": globalState.get("apiKey"),
     },
+    failOnStatusCode: false,
     body: requestBody,
   }).then((response) => {
     logRequestId(response.headers['x-request-id']);
 
+    expect(res_data.status).to.equal(response.status);
     expect(response.headers["content-type"]).to.include("application/json");
-    expect(response.body.payment_id).to.equal(payment_id);
-    expect(response.body.amount).to.equal(globalState.get("paymentAmount"));
-    // expect(response.body.amount_capturable).to.equal(0);
-    expect(response.body.amount_received).to.be.oneOf([0, null]);
-    expect(response.body.status).to.equal("cancelled");
+    if(response.body.payment_id !== undefined) {
```

Comment:
why are we have a condition on payment_id? why not on status code?

Reply:
it has been converted to status code